www/config.yml

```diff
@@ -933,6 +933,18 @@ chapters:
   twitter: Pyladiesowerri
   website: IN
   whatsapp: bit.ly/pyladiesimo
+- email: pyladies-bcn@googlegroups.com
```

Comment:
We require that all official `PyLadies` business be done via the official `@pyladies.com` email address.

Reply:
And as a community, how do we get an oficial barcelona@pyladies.com email address?

Reply:
Why, thought you'd never ask! There is this email actually already `bcn@pyladies.com`! Can you ping me in the PyLadies Slack to get this password reset - https://slackin.pyladies.com/?

Reply:
You can then register your chapter in the PyLadies chapter directory! See: https://www.pyladies.com/blog/Important-PyLadies-Update/important-pyladies-update/ about registration.

Reply:
Done! :D